pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.3.3</version>
+    <version>2.3.5-SNAPSHOT</version>
```

Comment:
This is the first of the checkstyle/findbugs PRs I'm looking at but I think we should get everyone to look at the parent changes first so we can release that and then all these PRs can use a non-SNAPSHOT version.